recipe/meta.yaml

```
@@ -7,7 +7,7 @@ package:
 
 source:
   url: https://github.com/conda/{{ name }}/archive/refs/tags/{{ version }}.tar.gz
-  sha256: 97938c7940777d014ded9b3031e8ae0bf516e68937ffb4f43075ea5ffccc1d24
+  sha256: 019a2893af9fb2e1aceed4ff66cabff35e0717195f25e90bed0dc1c119b4814a
 
 build:
   number: 0
```

While the previous build never went out, still think it is a good idea to bump the `build/number` to communicate there was an issue encountered with the previous build

```suggestion
  number: 1
```

---

fair enough, either works for me